main.py

```diff
@@ -1,7 +1,43 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre
+from db.models import Actor
 
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+    for genre in genres:
+        Genre.objects.create(
+            name=genre,
+        )
+    actors = [("George", "Klooney"), ("Kianu", "Reaves"),
+              ("Scarlett", "Keegan"), ("Will", "Smith"), ("Jaden", "Smith"),
+              ("Scarlett", "Johansson")]
```

Comment:
Place each tuple on a new line for a better readability

Reply:
Done